src/Admin/ConsultationReportAdmin.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+
+class ConsultationReportAdmin extends AbstractAdmin
+{
+    protected $datagridValues = [
+        '_page' => 1,
+        '_per_page' => 32,
+        '_sort_order' => 'ASC',
+        '_sort_by' => 'position',
+    ];
+
+    protected function configureShowFields(ShowMapper $showMapper)
+    {
+        $showMapper
+            ->add('name', null, [
+                'label' => 'Nom',
+            ])
+            ->add('url', UrlType::class, [
```

Comment:
maybe you wanted to use UrlType on `configureFormFields` instead?

Reply:
Thanks! Done.